GNUmakefile

```diff
@@ -38,4 +43,12 @@ test-compile:
 	fi
 	go test -c $(TEST) $(TESTARGS)
 
+build-dist:
+	GOOS=darwin GOARCH=amd64 go build -o "dist/darwin/amd64/$(BIN_NAME)"
+	GOOS=linux GOARCH=amd64 go build -o "dist/linux/amd64/$(BIN_NAME)"
+
+gh-dist: build-dist
+	aws s3 cp "dist/darwin/amd64/$(BIN_NAME)" "s3://grnhse-vpc-assets/terraform-plugins/darwin_amd64/$(BIN_NAME)"
```

Comment:
I would instead build the kernel name into the binary name so that when people download the file they know what they got. (My .02)

Reply:
I don't disagree, but I want to keep the name as similar as possible to the current distribution.

Reply:
ah, I see. that's an annoying choice they'be made :)